kolibri/core/assets/src/content/utils.js

```diff
@@ -0,0 +1,27 @@
+function validateAssessmentMetaData(data) {
+  // Data is coming from a serializer for a one to many key, so at least will return an empty array.
+  const assessmentMetaData = data.assessmentmetadata[0];
+  if (!assessmentMetaData) {
+    return {
+      assessment: false,
+    };
```

Comment:
Yes, it should be consistent - I thought I had added that. Will make the keys return across the board.